### PR TITLE
User friendship models

### PR DESCRIPTION
**Pull request by Evans-Munatsa**

### CODE REVIEW

**Comment by Evans-Munatsa** [comment]
the tests for UserFriendships

File: test/models/user_friendship_test.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class UserFriendshipTest < ActiveSupport::TestCase
+  should belong_to(:user)
+  should belong_to(:friend)
+
+  test "that creating a friendship works without raising an exception" do
```
Comment (ryannealmes) [inline test/models/user_friendship_test.rb]: Please fix the formatting here.

File: app/models/user_friendship.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class UserFriendship < ActiveRecord::Base
+	belongs_to :user
```
Comment (ryannealmes) [inline app/models/user_friendship.rb]: This does not look like it's indented the same as your other files. Please correct for readability.

File: test/models/user_test.rb [hunk 3]
```diff
@@ -1,11 +1,17 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
+
```
Comment (ryannealmes) [inline test/models/user_test.rb]: Whitespace.

File: test/models/user_test.rb [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
+
  def setup
     @user = User.new(name: "Example User", email: "user@example.com",
                      password: "foobar", password_confirmation: "foobar")
   end
```
Comment (ryannealmes) [inline test/models/user_test.rb]: Indentation.

File: test/models/user_test.rb [hunk 5]
```diff
@@ -1,11 +1,17 @@
...
 
+  should have_many(:user_friendships)
+  should have_many(:friends)
+
+
  test "should be valid" do
  	assert @user.valid?
```
Comment (ryannealmes) [inline test/models/user_test.rb]: Indentation.

**Pull request closed without merging**